File: test_wikidata.js
const url = 'https://query.wikidata.org/sparql?query=' + encodeURIComponent(
SELECT ?population WHERE {
  ?item rdfs:label "Delhi"@en.
  ?item wdt:P1082 ?population.
} LIMIT 1
) + '&format=json';

fetch(url, {headers: {'User-Agent': 'ResQ-Logix/1.0', 'Accept': 'application/sparql-results+json'}})
.then(r => r.json())
.then(r => console.log(JSON.stringify(r, null, 2)))
.catch(console.error);

File: test_wikidata.py
import urllib.request
import urllib.parse
import json

query = """
SELECT ?population WHERE {
  ?item rdfs:label "Delhi"@en.
  ?item wdt:P1082 ?population.
} LIMIT 1
"""
url = "https://query.wikidata.org/sparql?query=" + urllib.parse.quote(query) + "&format=json"

req = urllib.request.Request(url, headers={'User-Agent': 'ResQ-Logix/1.0', 'Accept': 'application/sparql-results+json'})
with urllib.request.urlopen(req) as response:
    print(response.read().decode())
